integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java

```diff
@@ -219,6 +220,21 @@ public void before(TestInfo testInfo) {
 
     managementApi.createCatalog(principalRoleName, catalog);
 
+    restCatalog = initCatalog(currentCatalogName, ImmutableMap.of());
+  }
+
+  @AfterEach
+  public void cleanUp() {
+    client.cleanUp(adminCredentials);
+  }
+
+  @Override
+  protected RESTCatalog catalog() {
+    return restCatalog;
+  }
+
+  @Override
+  protected RESTCatalog initCatalog(String catalogName, Map<String, String> additionalProperties) {
```

Comment:
Tests in the `integration-tests` module are supposed to be blackbox tests that run against a given Polaris Server. Why is this override necessary?